app/assets/javascripts/discourse/app/components/d-vh.gjs

```diff
@@ -0,0 +1,138 @@
+import Component from "@glimmer/component";
+import { cancel, scheduleOnce } from "@ember/runloop";
+import { service } from "@ember/service";
+import { clearAllBodyScrollLocks } from "discourse/lib/body-scroll-lock";
+import isZoomed from "discourse/lib/zoom-check";
+import discourseDebounce from "discourse-common/lib/debounce";
+import { bind } from "discourse-common/utils/decorators";
+
+const KEYBOARD_DETECT_THRESHOLD = 150;
+
+export default class DVh extends Component {
```

Comment:
`DVh` is a bit of a mysterious name, though. Might want something a bit more verbose for readability.

Reply:
DVirtualHeight